src/org/usfirst/frc/team2706/robot/commands/teleop/Climb.java

```diff
@@ -0,0 +1,28 @@
+package org.usfirst.frc.team2706.robot.commands.teleop;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
```

Comment:
Also, it may not be necessary to include the `@author` tag, but if you do want to keep it, use a more useful name, like your Github username. If you wanted to, you could also remove it, since it is auto-generated from Eclipse.